lib/vcs.ml

```diff
@@ -134,7 +134,7 @@ let git_describe ~dirty r commit_ish =
   let dirty = dirty && commit_ish = "HEAD" in
   let git_describe =
     Cmd.(
-      git_work_tree r % "describe" % "--always"
+      git_work_tree r % "describe" % "--always" % "--tags"
```

Comment:
Why is this needed? This allows including non annotated tags. That might be worth considering, it's been requested in the past, but I think it's unrelated to this bug fix.

Reply:
We have code where we parse unannotated tags and accept them while printing a warning, so I assumed it should be possible throughout. I can remove this but I feel like we should be consistent about unannotated tags.

Reply:
It would probably have made sense to make a separate PR for this but let's not bother!

I think it's worth having its own changelog entry though!